<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Escoja el modo de&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;conversi√≥n en el &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;submenu superior&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuValor">
    <property name="title">
     <string>Modo</string>
    </property>
    <addaction name="actionValor"/>
    <addaction name="actionResistencias_a_valor"/>
    <addaction name="actionColor"/>
   </widget>
   <addaction name="menuValor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionValor">
   <property name="text">
    <string>Resistencias a color</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Ley de Ohm</string>
   </property>
  </action>
  <action name="actionResistencias_a_valor">
   <property name="text">
    <string>Resistencias a valor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
